src/com/mraof/minestuck/network/CaptchaDeckPacket.java

```diff
@@ -120,6 +124,8 @@ public void execute(EntityPlayer player)
 				CaptchaDeckHandler.useItem((EntityPlayerMP) player);
 			else if(this.type == CAPTCHALOUGE && !player.getItemStackFromSlot(EntityEquipmentSlot.MAINHAND).isEmpty())
 				CaptchaDeckHandler.captchalougeItem((EntityPlayerMP) player);
+			else if(this.type == CAPTCHALOUGE_INV)
+				CaptchaDeckHandler.captchalougeInventoryItem((EntityPlayerMP) player);
```

Comment:
Unless the server side keeps track of which item that is mouseovered, you need to send that information with this packet.